backend/schema/database-schema.sql

```diff
@@ -62,4 +63,11 @@ CREATE TABLE IF NOT EXISTS teach (
 
     added_by TEXT REFERENCES auth_user(id),
     added_at TIMESTAMPTZ NOT NULL DEFAULT NOW()
+);
+
+CREATE TABLE IF NOT EXISTS study (
+    classroom_id    TEXT NOT NULL REFERENCES classroom(id) ON DELETE CASCADE,
+    user_id         TEXT NOT NULL REFERENCES auth_user(id) ON DELETE CASCADE,
+    section         INTEGER[] NOT NULL,
```

Comment:
Why is it an array? can users have multiple sections? But we also have a feature to create groups like in the class diagram.

Reply:
removed the section attribute